lib/seam/http_single_workspace.rb

```diff
@@ -2,7 +2,11 @@
 
 require_relative "request"
 require_relative "parse_options"
+require_relative "routes/resources/index"
+require_relative "routes/routes/index"
```

Comment:
@andrii-balitskyi This is a work around. We should remove these lines from there and lib/seam/http_multi_workspace.rb and move them into "routes/routes.rb` instead. Can you PR for that later? 🙏🏻